### PR TITLE
Generate sample sheets like the Illumina Experiment Manager does

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-2902

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/hot_pool.js [hunk 1]
```diff
@@ -307,45 +313,77 @@ HotTarget.pool = (function() {
...
+                     required: true,
+                     type: "select",
+                     getLabel: function(type) {
+                       return type.description;
+                     },
+                     values: Constants.illuminaExperimentTypes
+                   },
+                   {
+                     property: "sequencingParameters",
+                     label: "Instrument Model",
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/hot_pool.js]: This should say `Sequencing Parameters`

File: miso-web/src/main/webapp/scripts/hot_pool.js [hunk 2]
```diff
@@ -307,45 +313,77 @@ HotTarget.pool = (function() {
...
+                     label: 'Genome Folder',
+                     value: Constants.genomeFolder,
+                     required: true
+                   }, {
+                     property: "pools",
+                     label: "Lanes Configuration",
+                     required: true,
+                     type: "order",
+                     getLabel: Utils.array.getAlias,
+                     values: pools
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/hot_pool.js]: Just realized that this doesn't allow you to load two lanes of the same pool... it's only one lane per pool. That's probably a separate ticket though.

Comment (apmasell) [reply miso-web/src/main/webapp/scripts/hot_pool.js]: Yeah. It also can't deal with knowing about an S4 flowcell with joined/split lanes. For right now, they are only planning on using this on the MiSeq, so I'm okay with it being limited. I need to see sample sheets for the other platforms with those cases.

File: miso-web/src/main/webapp/scripts/hot_pool.js (hunk 1 above)
Comment (apmasell) [reply miso-web/src/main/webapp/scripts/hot_pool.js]: Fixed.

**Pull request merged**